data-manipulation/hashing/md5/hash-data-with-md5.yml

```diff
@@ -30,3 +31,7 @@ rule:
         - and:
           - number: 0x8003 = CALG_MD5
           - api: advapi32.CryptCreateHash
+      - and:
+        - format: dotnet
```

Comment:
gives `invalid rule: feature format(dotnet) not supported for scope function` since format is not allowed except for file scope. should we make it global?

Reply:
see rules.py:88

Reply:
The pull request https://github.com/mandiant/capa/pull/1093 adds the format feature to global scope.